indexer-app/src/cmr/indexer/data/concepts/granule.clj

```diff
@@ -233,5 +239,21 @@
 (defmethod es/parsed-concept->elastic-doc :granule
   [context concept umm-granule]
   (if (:deleted concept)
-    concept
+    (let [{:keys [concept-id revision-id deleted extra-fields native-id provider-id granule-ur
```

Comment:
There's no all revisions index for granules. We should not be indexing tombstones.